### PR TITLE
New DB2 integration

### PR DESCRIPTION
**Pull request by howardyoo**
DB2 integration contains detailed instructions in README, python script for running the integrations, and dashboard for viewing the integrations.

### CODE REVIEW

File: db2/wavefront_db2_metrics.py [hunk 1]
```diff
@@ -0,0 +1,521 @@
...
+out_type = "telegraf"
+
+# proxy setting which will be used when
+# wf_proxy settings
+proxy_host = "localhost"
+proxy_port = 2878
+
+# direct data ingestion settings
+# wf_direct
+direct_url = "https://longboard.wavefront.com"
```
Comment (akodali18) [inline db2/wavefront_db2_metrics.py]: longboard is a production cluster, so lets not give that as an example.
we can just say https://<INSTANCE>.wavefront.com

Comment (howardyoo) [reply db2/wavefront_db2_metrics.py]: OK

File: db2/wavefront_db2_metrics.py [hunk 2]
```diff
@@ -0,0 +1,521 @@
...
+
+
+def log(log_type, log_msg, host, db):
+    msg = None
+    if out_format == "wavefront":
+        msg = "{0}.integration.msg 1 source={1} database=\"{2}\" type=\"{3}\" msg=\"{4}\"".format(prefix,
+                                                                                                  host,
+                                                                                                  db,
+                                                                                                  log_type,
+                                                                                                  log_msg.replace(
```
Comment (akodali18) [inline db2/wavefront_db2_metrics.py]: formatting seems off.

File: db2/wavefront_db2_metrics.py [hunk 3]
```diff
@@ -0,0 +1,521 @@
...
+                                                                                    host,
+                                                                                    db,
+                                                                                    log_type,
+                                                                                    log_msg.replace(" ",
+                                                                                                    "\\ "))
+
+    if debug is False and msg is not None:
+        if out_type == "telegraf":
+            print msg
+
```
Comment (akodali18) [inline db2/wavefront_db2_metrics.py]: no line break required.

**Pull request merged**